mod/metrics/set/abstract/expanded_researched_details.rb

```diff
@@ -12,45 +12,48 @@ def credit_details
   end
 
   view :expanded_researched_details do
-    checked_by = card.fetch trait: :checked_by, new: {}
-    checked_by = nest(checked_by, view: :core)
     wrap_researched_details do
-      [
-        wrap_with(:div, checked_by, class: "double-check mt-3"),
-        wrap_with(:div, _render_sources, class: "cited-sources mt-3"),
-        overridden_calculated_value
-      ]
+      [checked_by_details, source_details, override_details]
     end
   end
 
-  def overridden_calculated_value
+  def checked_by_details
+    return if metric_card.designer_assessed?
```

Comment:
this reorganization was about making checked by details _not_ show up on designer assessed metrics.